src/main/java/Daisy.java

```diff
@@ -0,0 +1,52 @@
+import java.util.Scanner;
+
+public class Daisy {
+    public static void main(String[] args) {
```

Comment:
I like you named these literals. This improves the readability of your code as you're removing redundant lines